5-app-infra/README.md

```diff
@@ -0,0 +1,41 @@
+# 5-app-infra
+
+The purpose of this step is to set up an example app, deploying a Compute Engine instance in one of the business unit created on step 4-projects.
+This Compute Engine instance will be created using the base network created during step 3-networks to acess private services.
+
+## Prerequisites
+
+1. 0-bootstrap executed successfully.
+1. 1-org executed successfully.
+1. 2-environments executed successfully.
+1. 3-projects executed successfully.
+## Usage
+### Run terraform locally
```

Comment:
Will we be adding CB instructions by pushing [here](https://github.com/terraform-google-modules/terraform-example-foundation/blob/3c62c9e47961142ecd0d67ff6dea4e153cb541fc/4-projects/business_unit_1/shared/example_infra_pipeline.tf#L43) in a follow up?

Reply:
Yes